two_factor/forms.py

```diff
@@ -123,16 +126,21 @@ class AuthenticationTokenForm(OTPAuthenticationFormMixin, forms.Form):
     use_required_attribute = False
     idempotent = False
 
+    error_messages = {
+        'invalid_device_id': _('Device id is not valid.'),
+    }
+
     def __init__(self, user, initial_device, **kwargs):
         """
-        `initial_device` is either the user's default device, or the backup
-        device when the user chooses to enter a backup token. The token will
-        be verified against all devices, it is not limited to the given
-        device.
+        `initial_device` is either the user's default device a backup device
+        when the user chooses to enter a backup token.
```

Comment:
Not sure I understand this new comment

Reply:
I have updated the comment to include the crucial missing word "or" in "default device or a backup device"